GeneralsMD/Code/GameEngine/Source/GameNetwork/NetCommandMsg.cpp

```diff
@@ -759,6 +882,25 @@ void NetChatCommandMsg::setPlayerMask( Int playerMask )
 	m_playerMask = playerMask;
 }
 
+/**
+ * Get the byte count for this chat message.
+ */
+size_t NetChatCommandMsg::getPackedByteCount() const
+{
+	return sizeof(NetPacketChatCommand) + sizeof(UnsignedByte) + m_text.getByteCount() + sizeof(m_playerMask);
```

Comment:
Maybe also add an inline comment here to explain what this UnsignedByte is (it is not awfully obvious)